src/modules/displayTasks.js

```diff
@@ -0,0 +1,23 @@
+import { getLocalData } from './storage.js';
+
+const taskList = getLocalData();
+const listSection = document.querySelector('.tasks_list');
+
+const displayTasks = () => {
+  // const listSection = document.querySelector('.tasks_list');
```

Comment:
Kindly do not commit old pieces of code as inline comments. They will make your project look messy according to JavaScript best practices.